packages/xstate-react/src/useService.ts

```diff
@@ -1,39 +1,30 @@
 import { useMemo } from 'react';
-import { EventObject, State, Interpreter, Typestate } from 'xstate';
-import { useSubscription, Subscription } from 'use-subscription';
+import { EventObject, State, Interpreter, Typestate, Sender } from 'xstate';
+import { useActor } from './useActor';
+import { ActorRef } from './types';
+
+export function fromService<TContext, TEvent extends EventObject>(
+  service: Interpreter<TContext, any, TEvent, any>
+): ActorRef<TEvent, State<TContext, TEvent>> {
+  const { machine } = service;
+  return {
+    send: service.send.bind(service),
```

Comment:
similarly, this can trigger a warning when a child component sends to it from within first `useEffect`:
https://github.com/davidkpiano/xstate/blob/1273d72447af45b6c92da9b002f82619f566ebb7/packages/core/src/interpreter.ts#L572-L586

Maybe `@xstate/react` should always add `deferEvents: true` to the interpreter?